libproject/test/libprojecttest.cpp

```diff
@@ -298,6 +280,9 @@ class TestRemoveSubprojects : public TestSkeleton {
     case_6_pathToMainFileWhichWillNotBeLoaded,
     case_7_pathToMainFileWhereWillBeRemoveAndCountActions;
 
+    string case_0_contentBackup,
```

Comment:
#style
We didn't have supplied code style, but in most of styles that are presented - it's forbidden to declare variables in comma separated way.